test/jasmine/tests/parcoords_test.js

```diff
@@ -962,7 +970,11 @@ describe('parcoords basic use', function() {
         reversedMockCopy.data[0].dimensions.forEach(function(d) {d.id = 'R_' + d.id;});
         reversedMockCopy.data[0].dimensions.forEach(function(d) {d.label = 'R_' + d.label;});
 
-        Plotly.plot(gd, reversedMockCopy).then(function() {
+        Plotly.react(gd, mockCopy)
+        .then(function() {
+            return Plotly.plot(gd, reversedMockCopy);
```

Comment:
We should get rid of `Plotly.plot` in v2 - Etienne liked to use it in tests instead of `newPlot` when we know the div we're plotting into is new, as it saves some time purging previous items, esp. gl contexts IIRC. But if it's used like this on a div that already has a plot the effects are extremely confusing, and I don't think we've had a legitimate use of this behavior (appending its traces to the existing graph) for a long long time.

Reply:
do you think this is something we could just do or do we need to make more noise about it being deprecated somehow?

Reply:
Note: it looks like we have zero instances of `Plotly.plot()` in our current docs, so that suggests we could just drop it.

Reply:
Yep - `Plotly.plot` behaves exactly like `Plotly.newPlot` except when it does confusing things that are much better done with `Plotly.react`. Thanks for verifying that we don't use it in our docs!